x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_policy_create_multi_step_extension.tsx

```diff
@@ -6,13 +6,124 @@
  */
 
 import React, { memo } from 'react';
-import type { PackagePolicyCreateMultiStepExtensionComponentProps } from '@kbn/fleet-plugin/public';
+import {
+  EuiText,
+  EuiIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLink,
+  EuiPanel,
+  EuiSpacer,
+  useEuiTheme,
+} from '@elastic/eui';
+import styled from 'styled-components';
+import { css } from '@emotion/css';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useKibana } from '../../../../../common/lib/kibana';
+
+const CenteredEuiFlexItem = styled(EuiFlexItem)`
+  align-items: center;
+`;
 
 /**
- * TBD: A component to be displayed in multi step onboarding process.
+ * A component to be displayed in multi step onboarding process.
  */
-export const EndpointPolicyCreateMultiStepExtension =
-  memo<PackagePolicyCreateMultiStepExtensionComponentProps>(({ newPolicy }) => {
-    return <></>;
-  });
+export const EndpointPolicyCreateMultiStepExtension = memo(() => {
+  const { docLinks } = useKibana().services;
+  const { size } = useEuiTheme().euiTheme;
+
+  const title = (
+    <EuiText>
+      <h3>
+        <FormattedMessage
+          id="xpack.securitySolution.endpoint.policy.multiStepOnboarding.title"
+          defaultMessage="We'll save your integration with our recommended defaults."
+        />
+      </h3>
+    </EuiText>
+  );
+
+  const logoSize = `calc(2 * ${size.xxxxl})`;
+  const securityLogo = (
+    <EuiIcon
+      type="logoSecurity"
+      css={css`
+        width: ${logoSize};
+        height: ${logoSize};
+      `}
+    />
+  );
+
+  const features = (
+    <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false}>
+      <EuiFlexItem grow={false}>
+        <EuiIcon type="check" />
+      </EuiFlexItem>
+
+      <EuiFlexItem>
+        <EuiText size="m">
+          <p>
+            <FormattedMessage
+              id="xpack.securitySolution.endpoint.policy.multiStepOnboarding.feature"
+              defaultMessage="Windows, Mac, and Linux event collection"
```

Comment:
Nit: "macOS" (capitalized as such) is the proper name of the operating system, and Apple usually uses "Mac" to refer to a piece of hardware
```suggestion
              defaultMessage="Windows, macOS, and Linux event collection"
```